images/prow-tests/Dockerfile

```diff
@@ -59,13 +58,3 @@ RUN ko version > /ko_version
 # Extra tools through gem
 RUN gem install mixlib-config -v 2.2.4  # required because ruby is 2.1
 RUN gem install mdl -v 0.5  # required because later version of mdl requires ruby >= 2.4
-
```

Comment:
This can only be done when all repos relying on githubhelper are using latest test-infra. Do we know whether this is done already? How about release branches?

Reply:
You are right, it cannot be deleted as release branches are still having it. Reverted the change.
Thanks for pointing it out!